pytorch_lightning/callbacks/gpu_stats_monitor.py

```diff
@@ -117,7 +117,13 @@ def on_train_epoch_start(self, trainer, pl_module):
         self._snap_inter_step_time = None
 
     @rank_zero_only
-    def on_train_batch_start(self, trainer, pl_module, batch, batch_idx, dataloader_idx):
+    def on_train_batch_start(self, trainer, pl_module, *_):
+        if self._log_stats.intra_step_time:
+            self._snap_intra_step_time = time.time()
```

Comment:
where is this used?

Reply:
Here:
https://github.com/PyTorchLightning/pytorch-lightning/blob/5a3b9435f1b56f0841f527e39a4eb93b11da9df5/pytorch_lightning/callbacks/gpu_stats_monitor.py#L150